ironic/drivers/modules/drac/raid.py

```diff
@@ -767,7 +861,8 @@ def create_configuration(self, task,
                 'is_reboot_required']
             controllers.append(controller)
 
-        return _commit_to_controllers(node, controllers)
+        return _commit_to_controllers(node, controllers,
+                                      substep="virtual_disk_created")
```

Comment:
Instead of adding a new substep, let's make the default for substep be "complete" and not specify it here.

Reply:
Done